README.md

```diff
@@ -1,8 +1,11 @@
-<center>
-<img src="frontend/static/app-icon.png" width=144>
-</center>
+<p align="center">
+  <a href="https://hstspreload.org/">
+    <img src="frontend/static/app-icon.png" alt="" width="144" height="144">
```

Comment:
Personally I'll use the svg instead of the png.

Reply:
Indeed, that would be a good idea, since we're already using HTML for markup at this point. I'll merge this for now, though.